### PR TITLE
Small changes to the NFT migration guide

### PR DESCRIPTION
**Pull request by joshuahannan**

### CODE REVIEW

File: docs/cadence_migration_guide/nft-guide.mdx [hunk 1]
```diff
@@ -237,6 +237,9 @@ access(all) contract ExampleNFT: NonFungibleToken {
 
 This will ensure that your `NFT` resource has all the correct fields and functions.
 
+As part of this, you should remove the `NonFungibleToken.INFT` implementation specification
```
Comment (gregsantos) [inline docs/cadence_migration_guide/nft-guide.mdx:240]:
```suggestion
As part of this upgrade, you should remove the `NonFungibleToken.INFT` implementation specification
```

File: docs/cadence_migration_guide/nft-guide.mdx [hunk 2]
```diff
@@ -255,6 +258,16 @@ access(all) resource Collection: NonFungibleToken.Collection {
...
 as part of the Flow team's custom state migrations.
 
+### Change the access specification of `ownedNFTs` to `access(contract)`
+
+This one is EXTREMELY important. With the addition of entitlements and safe downcasting,
+now anyone with a reference to your collection can downcast it to its concrete type
+even if the reference is just of an interface type. This means that you have to be
+very careful about which fields and functions are declared as public.
+
+`ownedNFTs` used to be `access(all)`, but it should be changed to `access(contract)`.
```
Comment (gregsantos) [inline docs/cadence_migration_guide/nft-guide.mdx:268]: should we add a sentence that describes `access(contract)`?

**Review by gregsantos — APPROVED** [review]

**Comment by turbolent** [comment]
@joshuahannan Please also update https://github.com/onflow/cadence-lang.org/tree/main/versioned_docs/version-1.0/cadence_migration_guide

**Comment by joshuahannan** [comment]
sorry, I didn't realize we added it to the other version. I'll update the other one

**Pull request merged**